session-replay/Sources/DatadogSessionReplay/Recorder/Recorder.swift

```diff
@@ -45,7 +45,7 @@ internal class Recorder {
         self.init(
             configuration: configuration,
             rumContextObserver: rumContextObserver,
-            scheduler: MainThreadScheduler(interval: 0.25),
+            scheduler: MainThreadScheduler(interval: 0.1),
```

Comment:
This change is on purpose - `0.1s` frequency just looks better in the player produces more data for compression. We will work on finding the best value in dogfooding - for now consider this one arbitrary and temporary.